src/main/java/dbaccessor/user/UserAccessor.java

```diff
@@ -131,26 +129,20 @@ public boolean checkIfUsernameExists(String username) {
 
         String SQLStatment = "SELECT Username FROM User WHERE Username=@username";
         Statement statement = Statement.newBuilder(SQLStatment).bind("username").to(username).build();
-        List<User> resultSet = spannerTemplate.query(User.class, statement, new SpannerQueryOptions().setAllowPartialRead(true));
- 
-        return (!resultSet.isEmpty());
+        return !spannerTemplate.query(User.class, statement, new SpannerQueryOptions().setAllowPartialRead(true)).isEmpty();
     }
 
     public User getUser(long userId) {
 
         String SQLStatment = "SELECT * FROM User WHERE UserID=@userId";
         Statement statement = Statement.newBuilder(SQLStatment).bind("userId").to(userId).build();
-        List<User> resultSet = spannerTemplate.query(User.class, statement, null);
- 
-        return resultSet.get(0);
+        return spannerTemplate.query(User.class, statement, null).get(0);
     }
 
     public List<ListChats.UsernameChatId> getUsernameChatIdForSecondUsers(long userId) {
```

Comment:
This one however, should return an ImmutableList since the list is being returned.

Reply:
Done.